generic_config_updater/field_operation_validators.py

```diff
@@ -75,7 +79,7 @@ def _get_fields_in_patch():
 
         if 'value' in patch_element.keys() and isinstance(patch_element['value'], dict):
             for key in patch_element['value']:
-                cleaned_fields.append(cleaned_field+ '/' + key)
+                cleaned_fields.append(cleaned_field + '/' + key) if len(cleaned_field) > 0 else cleaned_fields.append(key)
```

Comment:
>if [](http://example.com/codeflow?start=65&length=2)

It's better to use if-else block. Single line is too long.

Reply:
Changed to if-else block